CHANGES.md

```diff
@@ -30,6 +31,7 @@ Change Log
 
 ##### Breaking Changes :mega:
 * Removed `Scene.copyGlobeDepth`. Globe depth will now be copied by default when supported. [#6393](https://github.com/AnalyticalGraphicsInc/cesium/pull/6393)
+* The default `classificationType` for `GroundPrimitive`, `CorridorGraphics`, `EllipseGraphics`, `PolygonGraphics` and `RectangleGraphics` is now `ClassificationType.TERRAIN`.  If you wish the geometry to color both terrain and 3D tiles, pass in the option `classificationType: Cesium.ClassificationType.BOTH`.
```

Comment:
Probably add a mention here about `BOTH` not being supported for ground geometry without materials?

Reply:
It's mentioned elsewhere